src/open-new-window.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+  <path stroke="#1F1F1F" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M20 13v7H4V4h7"/>
+  <path stroke="#1F1F1F" stroke-linecap="round" stroke-width="2" d="M12.4 11.6 20 4"/>
+  <path stroke="#1F1F1F" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M15.6 4H20v4.3"/>
```

Comment:
We default a lot of these attributes in our `Icon` component, so we could remove `stroke`, `stroke-linecap`, `stroke-linejoin` and `stroke-width` from all the `path`s.

Reply:
sorry I'm pretty new to SVGs, I did think this was likely the case but SVGOMG got grumpy when I tried to do that so I figured I must be wrong. will change now, thanks :)

Reply:
You could even go so far as to remove the decimals altogether as the whole numbers work well, and then you could even combine the paths to create a _tiny icon!_

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24"><path d="M20 13v7H4V4h7M12 12l8-8M16 4h4v4"/></svg>
```

Reply:
Don't apologise @kerryambt, I'm hyped that we're even talking about SVGs, I love em!

Reply:
I need to write something that explains what we can keep/remove from the attributes as we'll be expecting these icons to be used solely with our `Icon` component in the design system

Reply:
well I cant say I fully understand what you did with optimisation but that's nuts. I find it so mad that a whole icon can be just over 100 bytes. soooo small it's kinda mindblowing :o

you optimised that way more than SVGOMG did though, maybe you need to fork SVOMG and call it SVGTOMG XD

seriously though I think a little addition to the contributing part of the readme would be very helpful :)